proxies/live/apiproxy/policies/AssignMessage.MessageBatches.Create.Request.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.MessageBatches.Create.Request">
+    <DisplayName>AssignMessage.MessageBatches.Create.Request</DisplayName>    
+    <Properties/>
+    <AssignTo createNew="false" transport="http" type="request"/>
+    <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
+    <AssignVariable>
+        <Name>target.copy.pathsuffix</Name>
+        <Value>false</Value>
+    </AssignVariable>
+    <AssignVariable>
+        <Name>requestpath</Name>
+        <Value>/api/v1/send</Value>
+    </AssignVariable>
+    <Set>
+        <Payload contentType="application/json" variablePrefix="%" variableSuffix="#">%data.payload#</Payload>
+        <Headers>
+            <Header name="Content-Type">application/json</Header>
```

Comment:
I think either duplicate the AssignMessage.AuthenticationDetails set for the authentication header or make sure to run that policy before sending this request - auth will not work otherwise.